### DISCUSSION TITLE
15.2.5

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#702] [General]
## Releases Notes for 15.2.5
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.2.5/draw.io-15.2.5-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.2.5/draw.io-15.2.5-windows-no-installer.exe)
[macOS - Intel](https://github.com/jgraph/drawio-desktop/releases/download/v15.2.5/draw.io-x64-15.2.5.dmg)
[macOS - ARM](https://github.com/jgraph/drawio-desktop/releases/download/v15.2.5/draw.io-arm64-15.2.5.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v15.2.5/drawio-amd64-15.2.5.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v15.2.5/drawio-x86_64-15.2.5.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v15.2.5/drawio-x86_64-15.2.5.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates from Electron 12 to Electron 14. This provides a number of improvements in the security model, particularly around process isloation.
- Updates to [diagrams.net core 15.2.5](https://github.com/jgraph/drawio/blob/v15.2.5/ChangeLog). All changes from 15.1.4 to 15.2.5, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v15.2.5'>15.2.5</a>.</em>

**Comment by taozina0587** [comment]
I'm not able to change the theme. I select the dark theme and ask to restart the program. I do this! But still on the white theme. I've done the procedure more than 5 times and nothing

**Reply by davidjgraph** [reply]
Looks like we've lost access to localstorage in the update, we'll investigate

**Comment by davidjgraph** [comment]
15.2.7 is released with a fix for configuration load failure.
